### PR TITLE
developer quotes component

### PR DESCRIPTION
**Pull request by sean1588**
adds stencil component for testimonials page

### CODE REVIEW

File: stencil/src/components/developer-quotes/developer-quotes.tsx
Comment (zchase) [inline stencil/src/components/developer-quotes/developer-quotes.tsx:84]:
```suggestion
                                                <a class="flex flex-col" href={ this.enableLink(q.source) ? q.link : undefined } target="_blank" rel="noopener noreferrer">
```

I think we still have to include the `noopener noreferrer` tag because of older browsers: https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable

**Review by zchase — APPROVED** [review]
LGTM once the merge conflict is fixed up. Which is weird to me that it exists in the bundle file..

**Comment by sean1588** [comment]
> LGTM once the merge conflict is fixed up. Which is weird to me that it exists in the bundle file..

yeah, ive been getting these a lot lately with the bundle files... not sure why. do the bundle files even need to be committed, since i am assuming they get built in the pipeline anyway. i wonder if we can git ignore these.

**Pull request merged**
